rdkit/Chem/UnitTestCrippen.py

```diff
@@ -85,24 +84,23 @@ def testRepeat(self):
       clog = self.clogs[i]
       tmp = Crippen.MolLogP(mol)
       tmp = Crippen.MolLogP(mol)
-      self.assertTrue(feq(clog, tmp), 'bad logp fooutF,r %s: %4.4f != %4.4f' % (smi, clog, tmp))
+      self.assertTrue(feq(clog, tmp), f'bad logp for {smi}: {clog:4.4f} != {tmp:4.4f}')
```

Comment:
```
f'bad logp fooutF,r {smi}: {clog:4.4f} != {tmp:4.4f}'
```

Reply:
You're probably right with your change.

@greglandrum Why call `Crippen.MolLogP` twice?

Reply:
> ```
> f'bad logp fooutF,r {smi}: {clog:4.4f} != {tmp:4.4f}'
> ```

I don't think it is right. While other code call `bad logp` right but at this time not? Related to the second answer, the test named `testRepeat` when he want to guarantee the code does not change it behaviour if called twice, I guessed.

Reply:
Right. missed that. In this case, I would add an assert after each call.